CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-authn-token-http ChangeLog
 
+## 8.1.0 - 2023-06-08
+
+### Fixed
+- Uniform HTTP errors with invalid username hitting `/hash-parameters`.
+  Return fake hash parameters in case of invalid username.
+
```

Comment:
Should mention and/or refer to the new related config options.  In particular, it should be noted that `hmacKey` needs to be set to whatever it should be.